### PR TITLE
Update SSL trust issue

### PR DESCRIPTION
**Pull request by maxhelias**
Fix https://github.com/dunglas/symfony-docker/issues/79

### CODE REVIEW

**Comment by thePanz** [comment]
@maxhelias the mentioned command `sudo security add-trusted-cert ` is not recognized under Linux (do we need to install a separated tool for that?)

**Comment by maxhelias** [comment]
Yes I miss that 😅 
For linux, I think it should be like that : 
```
sudo cp ./docker/cert/local/root.crt /usr/local/share/ca-certificates/foo.crt && sudo update-ca-certificates
```

Can you confirm, please ?

File: docker/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Used for copy the self-signed certificate from caddy and add it to the trusted certificates
+# see https://github.com/dunglas/symfony-docker/blob/master/docs/troubleshooting.md#fix-chromebrave-ssl
+cert/*
```
Comment (dunglas) [inline docker/.gitignore:3]: Do we need to create a temporary directory? Can't we use a pipe or a temporary file?

**Comment by B-Galati** [comment]
What about using something like mkcert ?

**Comment by maxhelias** [comment]
@B-Galati, In fact Caddy already does the job but inside the container. The problem we have is that the certificate is not approved on the local machine.
Is it possible to copy it and trust it with mkcert?

**Comment by B-Galati** [comment]
@maxhelias I am not sure, that may be possible by copying the Caddy certificate at right place.

In the end, I think that it would be simpler to use a local certificate provided by `mkcert` than using the one provided by Caddy.

It would require to add a new volume with the generated certificates and use them in Caddy config.

File: docker/.gitignore (hunk 1 above)
Comment (maxhelias) [reply docker/.gitignore]: It's now in tmp. Are these command good ?

**Comment by maxhelias** [comment]
I re-open this PR in #93 .. I don't know why the conflict resolution was not updating here.

@B-Galati It could be a solution, feel free to open a issue or PR to have it in the pipeline. 
On the other hand, I don't see how we can do this while keeping the first idea of ​​the project which is to have an environment ready with simply "docker-compose up" (like api-platform distribution). But I would be happy if we can find an effective solution.

**Comment by B-Galati** [comment]
> On the other hand, I don't see how we can do this while keeping the first idea of ​​the project which is to have an environment ready with simply "docker-compose up" (like api-platform distribution). But I would be happy if we can find an effective solution.

@maxhelias Agree :heavy_check_mark: 

If it can help I wrote a script that install mkcert in the current directory if not installed globally: https://gist.github.com/B-Galati/eb1f58de5235a9abe4277eaf946fb85a

**Comment by thePanz** [comment]
@B-Galati have a look at #106 :)
I was able to get the mkcert certs working with Caddy, with no impact on the `production` environment :)

**Pull request closed without merging**